app/server/runtime/src/main/java/io/syndesis/server/runtime/DefaultMigrator.java

```diff
@@ -15,56 +15,74 @@
  */
 package io.syndesis.server.runtime;
 
-import java.io.IOException;
+import static io.syndesis.common.util.Json.map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.stereotype.Service;
 
 import javax.script.ScriptEngine;
 import javax.script.ScriptEngineManager;
 import javax.script.ScriptException;
 
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Paths;
+
 import io.syndesis.common.util.IOStreams;
 import io.syndesis.common.util.Resources;
 import io.syndesis.common.util.SyndesisServerException;
-import io.syndesis.server.jsondb.WithGlobalTransaction;
 import io.syndesis.server.jsondb.JsonDB;
+import io.syndesis.server.jsondb.WithGlobalTransaction;
 import io.syndesis.server.jsondb.dao.Migrator;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.core.io.Resource;
-import org.springframework.core.io.ResourceLoader;
-import org.springframework.stereotype.Service;
-
-import static io.syndesis.common.util.Json.map;
-
 @Service
 public class DefaultMigrator implements Migrator {
     private static final Logger LOG = LoggerFactory.getLogger(DefaultMigrator.class);
-
+    private String migrationDir;
```

Comment:
Since spring gives us abstraction over resources, I think there is no need for `migrationDir`. We just specify the resource paths (in spring resource syntax) that the `ResourceLoader` needs to look into.